File: CS/DataLayer/Data/Classes/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Collections;
using System.ComponentModel.DataAnnotations.Schema;

namespace DataLayer.Data.Classes
{
    public class Account {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public string AccountName { get; set; }
        public decimal Amount { get; set; }
    }
}

File: VB/DataLayer/Data/Classes/Account.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Collections
Imports System.ComponentModel.DataAnnotations.Schema

Namespace DataLayer.Data.Classes
    Public Class Account
        <DatabaseGenerated(DatabaseGeneratedOption.Identity)> _
        Public Property ID() As Integer
        Public Property AccountName() As String
        Public Property Amount() As Decimal
    End Class
End Namespace
